webpack.config.babel.js

```diff
@@ -94,6 +94,7 @@ export default {
       {from: "bootstrap.js"},
       {from: "webextension/**/*.ftl"},
       {from: "webextension/icons/*"},
+      {from: "icon.png"},
```

Comment:
Instead of symlinking `icon.png` to `webextension/icons/lock.png`, let's just copy the file there in webpack:

```js
{from: "webextension/icons/lock.png", to: "icon.png"}
```

Reply:
Yeah, good thinking. I had tried that, but the rule before it seemed to supersede the second rule. No matter what order I put two rules in, the copy couldn't run on the same file twice. Maybe I'm being dense and missed something, though.